astroplan/scheduling.py

```diff
@@ -299,14 +463,54 @@ def _make_schedule(self, blocks):
         return new_blocks, True
 
 
-class PriorityScheduler(Scheduler):
+class ScheduleScheduler(object):
```

Comment:
ScheduleScheduler is currently a "duplicate" of the Scheduler so that SequentialScheduler can run since it hasn't yet been changed to use the Schedule. Once that is written, they will be merged.